CMake/macros.cmake

```diff
@@ -290,7 +290,7 @@ ENDFUNCTION()
 # SHOULD_SUCCEED - whether the test is expected to pass or fail.  Expected failing tests will not have
 #                  the scalar tests added.
 
-FUNCTION(QMC_RUN_AND_CHECK BASE_NAME BASE_DIR PREFIX INPUT_FILE PROCS THREADS SCALAR_VALUES SERIES SHOULD_SUCCEED)
+FUNCTION(QMC_RUN_AND_CHECK BASE_NAME BASE_DIR PREFIX INPUT_FILE PROCS THREADS SHOULD_SUCCEED)
```

Comment:
Could you update the documentation of the function arguments above?   Some changes in the order, plus the added optional argument pairs for series and scalar values.